doc/main/sections/experiments.tex

```diff
@@ -0,0 +1,47 @@
+\section{Eksperymenty}
+W naszym projekcie przeprowadziliśmy walidacji krzyżowej dla parametru k równego 10 dla drzewa utworzonego przez algorytm ID3 oraz dla drzewa utworzonego prze C4.5.
+
+\subsection{Wyniki}
+
+
+
+\begin{table}[H]
+    \centering
+    \begin{tabular}{|c|c|}
+    \hline
+    Algorytm                & Dopasowanie         \\ \hline
+    ID3                     & 81,5\%              \\ \hline
+    C4.5                    & 81,4\%              \\ \hline
+    \end{tabular}
+    \caption{Wyniki walidacji krzyżowej dla akceptorów}
+    \label{tab:crossing}
+\end{table}
+
+\begin{table}[H]
+    \centering
+    \begin{tabular}{|c|c|}
+    \hline
+    Algorytm                & Dopasowanie         \\ \hline
+    ID3                     & 83,2\%              \\ \hline
+    C4.5                    & 81,7\%              \\ \hline
+    \end{tabular}
+    \caption{Wyniki walidacji krzyżowej dla donorów}
+    \label{tab:crossing}
+\end{table}
+
+Z wyników walidacji...
+
+\begin{figure}[H]
```

Comment:
nie wiem czy to błąd czy jak ale wykresy są puste

Reply:
wykresy są puste, bo teraz mamy jedno k. Zostawiłem to by nie szukać po internetach jak się wykresy w latexu wstawia xD

Reply:
przecież było choćby w ostatnim projekcie xD no, a bez sensu to zostawiać xD